gunicorn/workers/gaiohttp.py

```diff
@@ -7,7 +7,7 @@
 
 from gunicorn import util
 
-if sys.version_info >= (3, 4):
+if sys.version_info[0] >= 3:
```

Comment:
Since we are going to drop the if statement soon (when we drop the Python 2 support), I'd prefer leaving this as-is.

Reply:
✅